x/process/internal/keeper/keeper.go

```diff
@@ -38,7 +38,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 }
 
 // Create creates a new process.
-func (k *Keeper) Create(ctx sdk.Context, msg *types.MsgCreateProcess) (*processpb.Process, error) {
+func (k Keeper) Create(ctx sdk.Context, msg *types.MsgCreateProcess) (*processpb.Process, error) {
```

Comment:
I understand why: it's because we pass keeper to another keeper without pointer.. like comsos is doing.
I would suggest to continue like this for now and to put the function on all keeper with no pointer.

Reply:
let's continue to discussion on https://github.com/mesg-foundation/engine/issues/1639#issuecomment-586685993